diff -Nurp -x '*~' -x '*.orig' tmp/apps/nshlib/nsh_usbconsole.c src/apps/nshlib/nsh_usbconsole.c
--- tmp/apps/nshlib/nsh_usbconsole.c	2015-06-10 21:03:48.453512458 +0300
+++ src/apps/nshlib/nsh_usbconsole.c	2015-07-27 21:26:48.152161758 +0300
@@ -284,6 +284,8 @@ restart:
  *
  ****************************************************************************/
 
+extern int conn_main(int argc, char *argv[]);
+
 int nsh_consolemain(int argc, char *argv[])
 {
   FAR struct console_stdio_s *pstate = nsh_newconsole();
@@ -301,7 +303,8 @@ int nsh_consolemain(int argc, char *argv
 
 #if defined(CONFIG_PL2303) || defined(CONFIG_CDCACM)
 #ifdef CONFIG_CDCACM
-  ret = cdcacm_initialize(CONFIG_NSH_USBDEV_MINOR, NULL);
+  //ret = cdcacm_initialize(CONFIG_NSH_USBDEV_MINOR, NULL);
+  ret = conn_main(0, NULL);
 #else
   ret = usbdev_serialinitialize(CONFIG_NSH_USBDEV_MINOR);
 #endif
